README.md

```diff
@@ -1,25 +1,43 @@
-# Project I
+# Project II - Game of Life!
 
-Welcome! This repo contains helpful information and starter files for your project, just fork this repo and go to town!
+Welcome! This repo contains our recreation of Conway's Game of Life.
 
 ## Team members
 
-Write here the names, career path and email of the members of the team
-
-## Assignation 
-
-Please use this format for your User Stories assignations
-
-| User Stories     | John Doe | Jane Doe |
-| ---------------- | :--: | ---: |
-| Add US text here |  X   |      |
-| Add US text here |      |    X |
-| Add US text here |  X   |      |
+### Axcel Espinal
+Career path: Salesforce <br/>
+Email: axcel.espinal@itksquare.edu.mx
+### Gabriel Osorno
+Career path: JavaScript <br/>
+Email: gabriel.osorno@itksquare.edu.mx
+### Roberto Tello
+Career path: Flutter <br/>
+Email: roberto.tello@itksquare.edu.mx
+<br/><br/>
+
+## User Stories assignations 
+
+### User Stories:
+1. As a user, whenever I navigate to the website and see an empty grid with the option to let the user set up the initial configuration.
+2. If the game is paused I can resume it
+3. I can clear the board
+4. I can click a 'Randomize' button that will generate an initial configuration for the game to play and this can only be done when the games has not started or is paused.
+5. I can see the amount of generation has passed since the start of the game
+6. I can click a button that would set the game back to the initial state (how the website initially loads in)
+7. (Optional) Research know patterns that occurs in the Game Of Life and allow the  user to see those patterns.
+
+
+| User Stories | Axcel E | Gabriel O | Roberto T |
+| :----------: | :---: | :---: | :---: |
+|    US - 1    |       |   X   |       |
+|    US - 2    |   X   |       |       |
+|    US - 3    |   X   |       |       |
+|    US - 4    |       |   X   |       |
+|    US - 5    |       |       |   X   |
+|    US - 6    |       |       |   X   |
+|    US - 7    |       |       |       |
+<br/>
 
 # Live demo
 
-Please include here a link to your [live demo](url_here_please)
-
-# Requeriments
-You need to use the same version of node that we use in class and no external dependices can be used. This means no frameworks or libraries.
-
+Here is a link to our [live demo](https://axcelespinal-ksquare.github.io/project2_conway/)
```

Comment:
Add the /index.html to the link so it points to the html file once it's there

Reply:
I think it is not needed. It will do it automatically once the file is there.

Reply:
You are right. Thanks.